minidump-stackwalk/stackwalker.cc

```diff
@@ -575,7 +668,8 @@ int ConvertModulesToJSON(const ProcessState& process_state,
       main_module_index = module_sequence;
 
     Json::Value module_data;
-    module_data["filename"] = PathnameStripper::File(module->code_file());
+    string filename_leaf = PathnameStripper::File(module->code_file());
```

Comment:
It seems a little unfortunate to be keying this by just the module filename, but I would guess that in practice having multiple DLLs with the same filename loaded probably doesn't work anyway, or if it does is extremely unlikely to happen.

Reply:
Yeah it doesn't work.